packages/edit-site/src/components/sidebar/index.js

```diff
@@ -1,117 +1,81 @@
-			</SidebarScreenWrapper>
-			<SidebarScreenWrapper path="/navigation">
-				<SidebarNavigationScreenNavigationMenus />
-			</SidebarScreenWrapper>
-			<SidebarScreenWrapper path="/navigation/:postType/:postId">
-				<SidebarNavigationScreenNavigationMenu />
-			</SidebarScreenWrapper>
-			<SidebarScreenWrapper path="/wp_global_styles">
-				<SidebarNavigationScreenGlobalStyles />
-			</SidebarScreenWrapper>
-			<SidebarScreenWrapper path="/page">
-				<SidebarNavigationScreen
-					title={ __( 'Manage pages' ) }
-					content={ <DataViewsSidebarContent /> }
-				/>
-			</SidebarScreenWrapper>
-			<SidebarScreenWrapper path="/page/:postId">
-				<SidebarNavigationScreenPage />
-			</SidebarScreenWrapper>
-			<SidebarScreenWrapper path="/:postType(wp_template)">
-				<SidebarNavigationScreenTemplatesBrowse />
-			</SidebarScreenWrapper>
-			{ ! isMobileViewport && (
-				<SidebarScreenWrapper path="/patterns">
-					<SidebarNavigationScreenPatterns />
-				</SidebarScreenWrapper>
-			) }
-			<SidebarScreenWrapper path="/:postType(wp_template_part)/all">
-				<SidebarNavigationScreenTemplatesBrowse />
-			</SidebarScreenWrapper>
-			<SidebarScreenWrapper path="/:postType(wp_template_part|wp_block)/:postId">
-				<SidebarNavigationScreenPattern />
-			</SidebarScreenWrapper>
-			<SidebarScreenWrapper path="/:postType(wp_template)/:postId">
-				<SidebarNavigationScreenTemplate />
-			</SidebarScreenWrapper>
-		</>
-	);
-}
+	const navigate = useCallback( ( direction, backFocusSelector ) => {
+		setNavDirection( ( prevDir ) => ( {
+			navDirection: direction,
+			focusSelector:
+				direction === 'forward' && backFocusSelector
+					? backFocusSelector
+					: prevDir.focusSelector,
+		} ) );
+	}, [] );
+
+	const wrapperRef = useRef();
+	useEffect( () => {
+		let elementToFocus;
+		if ( navDirection === 'forward' ) {
+			const [ firstTabbable ] = focus.tabbable.find( wrapperRef.current );
+			elementToFocus = firstTabbable ?? wrapperRef.current;
+		} else if ( navDirection === 'back' && focusSelector ) {
+			elementToFocus = wrapperRef.current.querySelector( focusSelector );
+		}
+		elementToFocus?.focus();
+	}, [ navDirection, focusSelector ] );
 
-function Sidebar() {
-	const { params: urlParams } = useLocation();
-	const initialPath = useRef( getPathFromURL( urlParams ) );
+	const disableMotion = useReducedMotion();
+	const { initial, animate } = getAnim( navDirection );
 
 	return (
-		<>
-			<NavigatorProvider
-				className="edit-site-sidebar__content"
-				initialPath={ initialPath.current }
-			>
-				<SidebarScreens />
-			</NavigatorProvider>
-			<SaveHub />
-		</>
+		<SidebarNavigationContext.Provider value={ navigate }>
+			<div className="edit-site-sidebar__content">
+				<motion.div
+					ref={ wrapperRef }
+					key={ routeKey }
+					className="edit-site-sidebar__screen-wrapper"
+					initial={ ! disableMotion && initial }
+					animate={ ! disableMotion && animate }
+					transition={ { duration: 0.14 } }
```

Comment:
What do you think about using `ANIMATION_DURATION` for consistency? Or is there any special meaning behind `0.14`?